<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wybór figury</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Na jaką figurę wymienić pionka?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioQueen">
       <property name="text">
        <string>Hetman</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioKnight">
       <property name="text">
        <string>Koń</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBishop">
       <property name="text">
        <string>Goniec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioRook">
       <property name="text">
        <string>Wieża</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
